### PR TITLE
Case study coderyte fix

### PR DESCRIPTION
**Pull request by jen-p**

### CODE REVIEW

**Comment by craigmcginley** [comment]
Deploy preview for *loving-austin-5f2eff* ready!

Built with commit 73c1bf11e02f84bd1638b16e8901d0e0fe6dc99b

https://deploy-preview-109--loving-austin-5f2eff.netlify.com

File: src/case-studies/3m-coderyte.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ Orders quickly followed.
 
 <Divider />
 
-# Going Live
+##S Going Live
```
Comment (craigmcginley) [inline src/case-studies/3m-coderyte.md]: Extra 'S'

File: src/case-studies/3m-coderyte.md [hunk 2]
```diff
@@ -150,7 +150,7 @@ CodeRyte’s executives were thrilled to be rewarded for their hard work while h
   loop
 />
 
-<Quote quotee="Robert Schulz" quoteeSub="CodeRyte Inc.">GoInvo just handled out customers for us</Quote>
+<Quote quotee="Robert Schulz" quoteeSub="CodeRyte Inc.">GoInvo just handled out customers for us.</Quote>
```
Comment (craigmcginley) [inline src/case-studies/3m-coderyte.md]: "just handled out"? Meant to be "just handed out" or "just handled our"?

File: src/case-studies/3m-coderyte.md (hunk 1 above)
Comment (jen-p) [reply src/case-studies/3m-coderyte.md]: fixed

File: src/case-studies/3m-coderyte.md (hunk 2 above)
Comment (craigmcginley) [reply src/case-studies/3m-coderyte.md]: @jen-p what about this one?

**Pull request merged**